spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java

```diff
@@ -776,4 +786,66 @@ public void setTruststorePassword(String truststorePassword) {
 
 	}
 
+	@Validated
+	public class Jaas {
+
+		/**
+		 * True to enable JAAS.
+		 */
+		private boolean enabled;
+
+		/**
+		 * Login module; default: com.sun.security.auth.module.Krb5LoginModule.
+		 */
+		private String loginModule;
+
+		/**
+		 * Control flag; default REQUIRED.
+		 */
+		private KafkaJaasLoginModuleInitializer.ControlFlag controlFlag =
+				KafkaJaasLoginModuleInitializer.ControlFlag.REQUIRED;
+
+		/**
+		 * Options.
+		 */
+		private final Map<String, String> options = new HashMap<>();
+
+		public boolean isEnabled() {
+			return this.enabled;
+		}
+
+		public void setEnabled(boolean enabled) {
+			this.enabled = enabled;
+		}
+
+		@NotNull
+		public String getLoginModule() {
```

Comment:
This is not needed; the assertions are done on the underlying object that is configured with these properties.